src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -204,11 +204,11 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
 
   // Begin loading the .dll/.pdb/.wasm files, but don't block here. Let other loading processes run in parallel.
   const dotnetWasmResourceName = 'dotnet.wasm';
-  const assembliesBeingLoaded = resourceLoader.loadResources(resources.assembly, filename => `_framework/_bin/${filename}`, 'assembly');
```

Comment:
Do we care about keeping these paths in tact? It does break users that scripted a custom resource loader, but it's small enough to fix. I'd prefer to not have an intimate understanding of what the runtime pack looks like in our SDK.

Reply:
I'm OK with removing it. IMO, it makes more sense to resolve the path of the assembly at a higher level and keep this resource loader location-agnostic.

Reply:
I'm fine with the paths changing (if they're retaining at least the `_framework` prefix, since that's the bit we've regarded as indicative that the rest is private).

Custom resource loaders are meant not to rely on specific URL formats. That's why in the "load resource" callback we supply the resource type, resource name, and default URL all as separate args. People shouldn't assume the default URL matches any specific format.